client/src/components/App/App.jsx

```diff
@@ -163,22 +178,31 @@ function App() {
       setIsSignInLoading(true);
       const resp = await axios.post(`${API_BASE_URL}/users/login`, signIn);
       let user = resp.data;
+
```

Comment:
code clean up idea - create a new function called "sign in" that basically does all the setItems. 

The general rule of how to clean up code is to split it into smaller reusable functions.